tests/unit/test_utils.py

```diff
@@ -609,7 +609,7 @@ def test_run_big(root_logger):
 
 
 def test_get_distgit_handler():
-    for wrong_remotes in [[], ["blah"]]:
+    for _wrong_remotes in [[], ["blah"]]:
```

Comment:
This looks like a typo, `_` seems to be the right choice here as `_wrong_remotes` is as unused as `wrong_remotes` was.

Reply:
Yeah, but, why not replace the name with `_`, getting rid of the unused name completely? Why just prefix it, when it is, as you said, unused? It was good enough for two other loops, so why not this one? :) We don't need `_wrong_remotes` to stick around.

Reply:
Well, I don't have a horse in this race and am happy to change it to whatever, though the difference with other loops is that that `_` replaced a single letter character. 
In `for t in threads` I can see we're iterating threads on first glance even without the `t`, while here it might be beneficial to keep the word(s) for better readability.